src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$gap: 1px;
+$cell-size: 100px;
+$width: 7*($cell-size + $gap) + 20px;
```

```suggestion
$calendarWidth: 7 * ($cell-size + $gap) + 20px;
```
What is 7? Try to avoid magic numbers. Also, would be great to give more descriptive names